src/com/linkedin/parseq/Task.java

```diff
@@ -229,6 +229,7 @@
     ArgumentUtil.requireNotNull(func, "function");
     final Task<Task<R>> nested = map("map: " + desc, func);
     nested.getShallowTraceBuilder().setSystemHidden(true);
+    nested.getShallowTraceBuilder().setTaskType("nestedTask");
```

Comment:
I think we should drop redundant "Task" suffix from the task types e.g.:
"nestedTask" -> "nested"
"withSideEffectTask" -> "withSideEffect"
"timeoutTask" -> "timeout"
"withTimeoutTask" -> "withTimeout"